src/api-reference/financial-integration/v4.financial-integration.markdown

```diff
@@ -26,19 +26,26 @@ Below are some benefits for using the Financial Integration service:
 * [Get Financial Transactions](#get-transactions)
 * [Post Financial Transactions Acknowledgements](#post-acknowledgements)
 * [Post Financial Transactions Confirmations](#post-confirmations)
+* [Post Financial Payment Confirmations](#payment-confirmations)
 * [Service Codes](#service-codes)
 * [Schema](#schema)
   * [Financial Documents](#schema-fidocuments)
-  * [Financial Document](#schema-fidocument)
-  * [Acknowledge Request](#schema-acknowledgerequest)
-  * [Acknowledge Response](#schema-acknowledgeresponse)
-  * [Acknowledge Response Item](#schema-acknowledgeresponse-item)
+  * [FIDocument](#schema-FIDocument)
+  * [AcknowledgeRequest](#schema-AcknowledgeRequest)
+  * [AcknowledgeResponse](#schema-AcknowledgeResponse)
+  * [AcknowledgeResponseItem](#schema-AcknowledgeResponseItem)
   * [Confirmation Request](#schema-confirmationrequest)
   * [Posting Confirmation Request Item](#schema-postingrequest)
   * [Posting Documents Details](#schema-postingdocs)
   * [System Messages](#schema-systemmsg)
   * [Posting Confirmation Response](#schema-postingconfirmationresponse)
-  * [Posting Confirmation Response Item](#schema-postingresponseitem)
+  * [PostingConfirmationResponseItem](#schema-PostingConfirmationResponseItem)
+  * [Payment Request](#schema-paymentrequest)
```

Comment:
This should be Payment Confirmation Request, but it's not included in the Swagger so I don't know the exact format for the name.

Reply:
edited. please check if this is what you actually meant.